profiles/tests/test_models.py

```diff
@@ -489,6 +489,10 @@ def test_ssn(self):
 
         self.fails_validation(sensitive_data)
 
+        for intermediary in "+-ABCDEFUVWXX":
+            sensitive_data.ssn = f"150977{intermediary}5554"
+            self.passes_validation(sensitive_data)
```

Comment:
The validator is most thoroughly tested in `profiles/tests/test_validators.py`. Those tests could be enhanced to include these new accepted characters. These other tests just ensure that the validator is used in the correct places, but they don't need to cover more than one case.